extensions/data-plane-selector/store/sql/data-plane-instance-store-sql/src/main/java/org/eclipse/dataspaceconnector/dataplane/selector/store/sql/SqlDataPlaneInstanceStoreExtension.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) 2020 - 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.dataplane.selector.store.sql;
+
+import org.eclipse.dataspaceconnector.dataplane.selector.store.DataPlaneInstanceStore;
+import org.eclipse.dataspaceconnector.dataplane.selector.store.sql.schema.DataPlaneInstanceStatements;
+import org.eclipse.dataspaceconnector.dataplane.selector.store.sql.schema.postgres.PostgresDataPlaneInstanceStatements;
+import org.eclipse.dataspaceconnector.runtime.metamodel.annotation.EdcSetting;
+import org.eclipse.dataspaceconnector.runtime.metamodel.annotation.Extension;
+import org.eclipse.dataspaceconnector.runtime.metamodel.annotation.Inject;
+import org.eclipse.dataspaceconnector.runtime.metamodel.annotation.Provider;
+import org.eclipse.dataspaceconnector.runtime.metamodel.annotation.Provides;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+import org.eclipse.dataspaceconnector.spi.transaction.TransactionContext;
+import org.eclipse.dataspaceconnector.spi.transaction.datasource.DataSourceRegistry;
+
+/**
+ * Extensions that expose an implementation of {@link DataPlaneInstanceStore} that uses SQL as backend storage
+ */
+@Provides(DataPlaneInstanceStore.class)
+@Extension(value = SqlDataPlaneInstanceStoreExtension.NAME)
+public class SqlDataPlaneInstanceStoreExtension implements ServiceExtension {
+    public static final String NAME = "Sql Data Plane Instance Store";
+
+    /**
+     * Name of the datasource to use for accessing data plane instances.
```

Comment:
This comment should be moved to the `value` field of the `@EdcSetting` annotation.